test/Microsoft.Azure.WebJobs.Host.UnitTests/ConverterManagerTests.cs

```diff
@@ -514,6 +514,28 @@ public void TestPatternMatcher_ResolveGenerics()
                 PatternMatcher.ResolveGenerics(int1, genArgs));
         }
 
+        public class TestConverter :
+            IConverter<Attribute, IAsyncCollector<string>>, // binding rule converter
+            IConverter<string, byte[]> // general type converter
+        {
+            public IAsyncCollector<string> Convert(Attribute input)
+            {
+                return null;
+            }
+
+            byte[] IConverter<string, byte[]>.Convert(string input)
+            {
+                return null;
+            }
+        }
+
+        [Fact]
+        public void PatternMatcher_Succeeds_WhenBindingRuleConverterExists()
+        {
+            var pm = PatternMatcher.New(typeof(TestConverter));
+            var func = pm.TryGetConverterFunc(typeof(string), typeof(byte[]));
+        }
```

Comment:
can you add a call for the other rule too: 
   pm.TryGetConverterFunc(typeof(Attribute), typeof(IAsyncCollector<string>); 
and assert they're non-null.